installer/k8s/README.md

```diff
@@ -168,7 +169,7 @@ rm -rf ${HOME}/streampipes-k8s
 | external.couchdb.appName                        | CouchDB application name                                 | "couchdb"                                |
 | external.couchdb.version                        | CouchDB version                                          | 3.3.1                                    |
 | external.couchdb.user                           | CouchDB admin username                                   | "admin"                                  |
-| external.couchdb.password                       | CouchDB admin password                                   | empty (auto-generated)                   |
+| external.couchdb.password                       | CouchDB admin password                                   | "admin"                                  |
```

Comment:
Should we leave the CouchDB default as `auto-generated`?

Reply:
already modified